common/client-libs/validator-client/src/nymd/fee/gas_price.rs

```diff
@@ -71,11 +71,9 @@ impl FromStr for GasPrice {
     }
 }
 
-impl Default for GasPrice {
-    fn default() -> Self {
-        format!("{}{}", defaults::GAS_PRICE_AMOUNT, defaults::DENOM)
-            .parse()
-            .unwrap()
+impl GasPrice {
+    pub fn default(denom: String) -> Result<Self, NymdError> {
```

Comment:
Perhaps it shouldn't be called `default` anymore given that it requires some arguments for construction?

Reply:
Yes, although the gas price in itself is still the default one. I'll see what better name I can come up with